rental.rb

```diff
@@ -0,0 +1,11 @@
+require './book'
+require './person'
+class Rental
+  attr_accessor :book, :person, :date
+
+  def initialize(book, person, date)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
From the requirements: 
a-Create the has-many side of Book and Rental (a book has many rentals).
a-Create the belongs-to side of Rental and Book (a rental belongs to a book).

b-Create the has-many side of Person and Rental (a person has many rentals).
b-Create the belongs-to side of Rental and Person (a rental belongs to a person)

Therefore I would advise you to add these codes: 
```ruby
    book.rentals << self  #The rental is adding itself to the @book achieving the 'belongs-to' association
```
```ruby
    person.rentals << self    #The rental is adding itself to the @person the 'belongs-to' association
```